app/assets/stylesheets/components/_icon.scss

```diff
@@ -36,3 +36,28 @@ $icon-min-padding: 2px;
     }
   }
 }
+
+// Login Specific Icons (non-uswds)
+.lg-icon {
```

Comment:
My thinking with the suggestion of "alert icon" in https://github.com/18F/identity-idp/pull/6983#discussion_r976594945 was to help disambiguate these icons from the USWDS icons, which are customized not just in the contents of the icon itself, but in the overall appearance and semantics of the icon usage, where a [design system icon](https://designsystem.digital.gov/components/icon/) is just the icon graphic itself.